### DISCUSSION TITLE
Is it possible to exclude only response

### DISCUSSION THREAD

**Discussion by aykutyuksek** [zalando/logbook#1655] [Q&A]
Hello,
I have a spring boot project. I want to log all requests but not all responses. Is there any way that i can exclude only responses in logbook?
Thanks,

**Comment by whiskeysierra** [comment]
A custom strategy would give you that. Check the readme for some details
and examples.

**Reply by aykutyuksek** [reply]
i checked readme, but i couldnt find something that i want clearly. should i use responseFilter for this?

**Comment by whiskeysierra** [comment]
https://github.com/zalando/logbook#strategy
